components/nav/nav.sass

```diff
@@ -0,0 +1,26 @@
+.nav 
+  height: 15vh
+  background: $dark-background
```

Comment:
So dark-background doesn't describe a purpose so much as it describes the color. So nav-background would be a more descriptive name regarding what that variable is used for.